gen-ai/sample-claude-multimodal.json

```diff
@@ -0,0 +1,108 @@
+{
+  "variable": [
+    {
+      "value": "https://api.anthropic.com/v1/messages",
+      "key": "api.url"
+    },
+    {
+      "value": "claude-3-opus-20240229",
+      "key": "llm"
+    },
+    {
+      "value": "<your-prompt-here>",
+      "current": "",
+      "key": "textInput"
+    },
+    {
+      "value": "<image-in-base64>",
+      "current": "",
+      "key": "imgBase64"
+    },
+    {
+      "value": "0",
+      "current": "",
+      "key": "temp"
+    },
+    {
+      "value": "<your-api-key>",
```

Comment:
is it worth specifying that this is your anthropic API key so it's not confused with mabl API key?

Reply:
Good point. Maybe `<anthropic-api-key>`?